Harvest.Core/Services/InvoiceService.cs

```diff
@@ -139,14 +141,14 @@ private async Task CreateMonthlyAcreageExpense(Project project)
 
         }
 
-        public async Task<int> CreateInvoices()
+        public async Task<int> CreateInvoices(bool manualOverride = false)
         {
-            var activeProjects = await _dbContext.Projects.Where(a => a.IsActive && a.Status == Project.Statuses.Active)
+            var activeProjects = await _dbContext.Projects.Where(a => a.IsActive && a.Status == Project.Statuses.Active && a.Accounts.Any())

```

Comment:
If the project is active, but there are no accounts I think we should throw an exception.